core/modules/language/tests/language.test

```diff
@@ -57,7 +57,7 @@ class LanguageListTest extends BackdropWebTestCase {
     $edit = array(
       'site_default' => $langcode,
     );
-    $this->backdropPost(NULL, $edit, t('Save configuration'));
+    $this->backdropPost(NULL, $edit, t('Save order'));
```

Comment:
...same as earlier, I think that we should keep "Save configuration".

Reply:
thanks :)